### PR TITLE
Add support for video backgrounds in the BackgroundImagePanel component

### PR DESCRIPTION
**Pull request by richtabor**

### CODE REVIEW

File: src/components/background/classes.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function BackgroundClasses( attributes ) {
 		{ 'has-background': attributes.backgroundColor || attributes.customBackgroundColor },
 		{ [ backgroundClass ]: backgroundClass },
 		{ 'has-parallax': attributes.backgroundImg && attributes.hasParallax },
-		{ 'has-background-image': attributes.backgroundImg },
+		{ [ `has-background-${ attributes.backgroundType }` ] : attributes.backgroundImg && attributes.backgroundType },
```
Comment (richtabor) [inline src/components/background/classes.js]: @phpbits Will this cause deprecation issues for folks running older versions of CoBlocks, before `backgroundType` existed?

File: src/blocks/hero/components/edit.js [hunk 2]
```diff
@@ -162,6 +163,11 @@ class Edit extends Component {
 					className={ classes }
 				>
 					<div className={ innerClasses } style={ innerStyles } >
+						{ backgroundType == 'video' ? 
+							<div className="coblocks-video-background">
+								<video playsinline="" autoplay="" muted="" loop="" src={ backgroundImg } ></video>
```
Comment (richtabor) [inline src/blocks/hero/components/edit.js]: @phpbits What do you think about making `autoplay`, `muted`, and `loop` ToggleControl options within the Background Settings panel (only if the backgroundType is 'video').

Comment (richtabor) [reply src/blocks/hero/components/edit.js]: On second though, just muted and loop - not autoplay (as it should autoplay).

File: src/components/background/classes.js (hunk 1 above)
Comment (phpbits) [reply src/components/background/classes.js]: @richtabor I've actually changed this to `has-background-${ attributes.backgroundType }` 👍

**Comment by richtabor** [comment]
Closes #151

**Pull request merged**